### PR TITLE
Fix #1138: Add research organizations on landing page

### LINKED ISSUE #1138: Add list of Research Organizations on landing page
Add the following organizations on the landing page:

- Facebook AI Research
- Georgia Tech
- Virginia Tech

### PR DESCRIPTION
**Pull request by deshraj**
@aka-jain Can you please check if the below screenshot looks good? Fixes #1138 

screencapture-localhost-8888-1502184729286

### CODE REVIEW

**Comment by aayusharora** [comment]
Hey @deshraj Can you please use webP images, instead of png.

File: frontend/src/views/web/landing.html [hunk 1]
```diff
@@ -80,26 +80,37 @@ <h4 class="grad-span">
...
+                        <div class="align-center">
+                            <h4 class="mons-med-font"><div>Who uses EvalAI?</h4>
+                            <p><span class="text-dark-black w-500 mons-med-font">Several Research Organizations have already started hosting their challenges on EvalAI. Some of them are</span>
+                            </p>
+                        </div>
+                    </div>
+                </div>
+                <div class="row center">
+                    <div class="col s1"></div>
+                    <div class="col s3 center">
```
Comment (aayusharora) [inline frontend/src/views/web/landing.html]: I would suggest to take away `width:100%` from here and add it to .scss by making an image class.
Also add `margin` property to the class.

File: frontend/src/views/web/landing.html [hunk 2]
```diff
@@ -80,26 +80,37 @@ <h4 class="grad-span">
...
+                    <div class="col s12">
+                        <div class="align-center">
+                            <h4 class="mons-med-font"><div>Who uses EvalAI?</h4>
+                            <p><span class="text-dark-black w-500 mons-med-font">Several Research Organizations have already started hosting their challenges on EvalAI. Some of them are</span>
+                            </p>
+                        </div>
+                    </div>
+                </div>
+                <div class="row center">
+                    <div class="col s1"></div>
```
Comment (aayusharora) [inline frontend/src/views/web/landing.html]: Please check what this `  <div class="col s1"></div>` is for? This seems extra code.

**Comment by aayusharora** [comment]
@deshraj Please check the above comments.

**Comment by the-dagger** [comment]
@deshraj it's better if all the images have a central gravity. 
Right now the FAIR logo looks out of place. 
If the latter 2 images have a central gravity then it would look a lot better. 
Please check the attacked image below.

**Comment by deshraj** [comment]
@the-dagger would you mind sharing the code for the same?

**Comment by aka-jain** [comment]
@aayusharora webP doesn't have all browsers support, so it's not good to use webP here, also png is good enough for the usage according to size and quality wise.

@deshraj Design doesn't look good to me, give me some time I'll share the bottom design soon.

**Comment by deshraj** [comment]
> @deshraj Design doesn't look good to me, give me some time I'll share the bottom design soon.

@aka-jain sounds good to me. Please feel free to add changes in the `landing` branch.

**Comment by the-dagger** [comment]
@deshraj the code is over here ,

[https://github.com/the-dagger/EvalAI/tree/landingHar](https://github.com/the-dagger/EvalAI/tree/landingHar)

I made some more modifications and came up with this,

screen shot 2017-08-08 at 3 37 47 pm

**Comment by aayusharora** [comment]
@aka-jain serving WebP if it is supported else serving the png using `source` tag should work here.

**Comment by aayusharora** [comment]
@deshraj @aka-jain by doing this,  we will save a lot of Kb's while requesting the Network. 
Obviously we are restricted to use png if the browser is not supporting WebP.

**Comment by deshraj** [comment]
@aayusharora sure sounds good. I will make the changes in a single go when @aka-jain shares the design.

**Comment by aka-jain** [comment]
@deshraj you can proceed, I'll make the changes in a different branch wherever needed.

**Comment by deshraj** [comment]
> @deshraj @aka-jain by doing this, we will save a lot of Kb's while requesting the Network. Obviously we are restricted to use png if the browser is not supporting WebP.

@aayusharora: currently we are moving with adding the images in the png format only. We will move all the static files to CDN sometime soon and that will reduce the number of requests to the server.

**Comment by aka-jain** [comment]
@deshraj @the-dagger please refer #1167 for this.

**Comment by deshraj** [comment]
Already fixed in #1167. Closing this.

**Pull request closed without merging**